storops/unity/resource/system.py

```diff
@@ -498,6 +505,75 @@ def upload_license(self, license):
         resp = self._cli.rest_post('/upload/license', files=files)
         resp.raise_if_err()
 
+    def get_remote_system(self, _id=None, name=None, **filters):
+        return self._get_unity_rsc(
+            UnityRemoteSystemList, _id=_id, name=name, **filters)
+
+    def create_remote_system(self, management_address,
```

Comment:
Remote system should be created manually in Unisphere in advance, shouldn't it?

Reply:
Yes, remote system is created manually beforehand. The interface `create_remote_system` provided in storops isn't used in Cinder driver.